docs/source/model_design/general_research.rst

```diff
@@ -888,9 +888,125 @@ respiratory disease.
+outstaning issue related to reponse is biased reponse, where individuals are 
+more likely to answer the questions or complete the surveys with a paticular 
+characteristic or a set of characteristics. Bias will introduce when certain 
+characteristic in questionnaire is related to probability of having a disease 
+or outcome. For example, you might observe a high response rate among elder 
+people and those unemployed by conducting a door-to-door interview for a disease. 
+The results are biased because we are accidentally assume these two groups are 
+actually experiencing higher risk of disease but ignore the fact that they are 
+more likey to be stay in home during the daytime.
+
+
+Advantages and disadvantages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cross-sectional study is less expensive and take shorter time to conduct than a 
+longitudinal study because the subjects are not necessarily to be tracked over 
+time, therefore the analysis of the study can start immediately after the data 
+collection finished. They are powerful in health planning, understadning etiology 
+and generation of hypotheses. Researchers can collect data about their 
+hypothesized phenomenon, and also gather the information about the demographic 
+and other relevant characteristics in order to compare the finidngs from 
+different groups. In addition, cross-sectional studies do not have a concern of 
+loss to follow-up.
+
+There are some disadvantages in cross-sectional study design. It's often diffcult 
+to draw a conclusion on causation because data are captured at single timepoint. 
+Researchers who chose a cross-sectional study design may be confounded by 
+historical factors during or before data collection. For example, a researcher 
+studying emergency preparedness may get flawed results if he conducted a survey 
+immediately after a major hurricane. In the same condition, a longitudinal study 
+would show trends in emergency preparedness and demonstrate if the hurricane had 
+an effect on the phenomenon. We may see the prevalence-incidence bias in 
+cross-sectional studies, especially in the case of longer-lasting diseases (e.g. 
+chronic diseases like diabetes or hypertension), any risk factor that results in 
+death will be under-represented among those with condition.
+
+
+Data analysis
+~~~~~~~~~~~~~
+
+For calculation of point/period prevalence and the prevalence ratio, we assumed 
+that both the exposures and the outcome were discrete variables. For example, 
+you would expect a binary distirbution for prevalence of smoking. "Yes, you are
+current smoker" or "No, you are current non-smoker". Sometimes we have observed 
+continuous distirbution for prevalence of smoking, the prevalence measure has to 
+fall below/above a certain threshold (e.g. 20 pack-years) or in a certain range 
+(e.g. 20 to 30 pack-years). In this circumstance, your analytical approach would 
+be to compare mean or median values in exposed population against unexposed 
```

Comment:
Could you discuss the implications of choosing different cutoffs within a distribution?